src/js/views/globals/root_views.js

```diff
@@ -306,10 +306,10 @@ const RootView = CollectionView.extend({
     // Add lowest layer (z-index) to highest
     this.addChildView(this.appView);
     this.addRegionView('tooltip', new TooltipRegionView({ $body }));
-    this.addRegionView('alert', new TopRegionView());
     this.addRegionView('modalSidebar', new ModalSidebarRegionView({ $body }));
     this.addRegionView('modal', new ModalRegionView({ $body }));
     this.addRegionView('modalSmall', new ModalRegionView({ $body }));
+    this.addRegionView('alert', new TopRegionView());
```

Comment:
alerts were rendering below the modal and darkening.. I can't think of a reason why that should be true.